### ISSUE TITLE
agent.host 설정 후 client에서 확인이 안되는 경우 문의

### ISSUE THREAD

**Issue by rindol** [scouter-project/scouter#546]
A서버에 Collector server가 구동되고 있고 B서버에서 agent.host를 기동하였습니다만,
client 에서 collect server 로그인은 되는데 object에서 B서버가 보이지가 않습니다.

B서버에서 A서버로 6100(tcp/udp) 포트 접속 여부는 확인하였습니다.

혹시 다른 부분에서 확인해봐야 하는 부분이 있을까요?
몇 시간째 찾아봤는데, 해결책을 찾지못해서 문의 드립니다.

참고로 A서버에서 agent.host 기동 시, A서버 Collector Server 로 수집이 잘 이루어지며 Client를 통해서도 A서버의 agent.host 정보가 확인됩니다.

버전은 모두 최신 버전입니다.

**Comment by gunlee01** [comment]
혹시 B서버에서 A서버로 접속 설정은 하셨나요? 
`net_collector_ip`

**Comment by rindol** [comment]
네 IP 및 PORT 모두 설정하였습니다. B서버에서 A서버로  nc로 6100 포트 열려 있는 건 확인하였습니다.

**Comment by rindol** [comment]
참고가 될지 모르겠습니다만, host 쪽 로그 첨부해드립니다.
-------------------
20190111 16:23:16 Version 2.5.1 2018-12-16 08:55 GMT
20190111 16:23:16 [A113] Counter Collector Started (#5)
20190111 16:23:17 [A10] host agent boot seed=x2m8gu3i1pk
20190111 16:23:17 [A01] Scouter Host Agent Version 2.5.1 2018-12-16 08:55 GMT
20190111 16:23:17 [A119] Agent UDP local.port=0
20190111 16:23:18 [INFO] Num of Cpu Cores : 8

**Comment by hymn-fly** [comment]
저도 비슷한 문제를 겪은바가 있어 공유합니다.
저의 경우는 AWS EC2를 이용하여 진행하였고
A 서버에 host agent 실행, B 서버에 collector server 실행 후,
A 서버에서 B 서버의 6100 포트에 접근 가능하도록 인바운드 설정해주었습니다. (TCP 프로토콜)

그러나 B 서버의 collector server를 바라보도록 client 를 실행시키니 server는 잘 인식되지만,
host agent가 잡히지 않았습니다.

저의 경우 원인은 UDP 프로토콜에 대해 6100 번 포트를 열어주지 않아서 host agent가 client에서 보이지 않았습니다.

netcat으로 접근테스트는 TCP만 열려있으면 접근이 잘 되어서 UDP에 대한건 놓치게 되네요.
TCP, UDP 두 프로토콜에 대해 6100번 포트 둘다 접근가능해야 합니다.
